src/aiida/tools/archive/imports.py

```diff
@@ -924,7 +924,7 @@ def _import_links(
                     raise ImportUniquenessError(
                         f'Node {in_id} already has an outgoing {link_type.value!r} link with label {link_label!r}'
                     )
-                if 'out_id' in link_uniqueness and out_id in existing_out_id_label:
+                if 'out_id' in link_uniqueness and out_id in existing_out_id:
```

Comment:
This was a genuine bug! Please see the surrounding code to understand the fix.

Also note how the `--pretty` mypy output is helpful here!

```
src/aiida/tools/archive/imports.py:927: error: Non-overlapping container check (element type: "int", container item type: "tuple[Any, Any]")  [comparison-overlap]
    ...              if 'out_id' in link_uniqueness and out_id in existing_ou...
                                                        ^~~~~~~~~~~~~~~~~~~~~...
```